src/responsibleai/docker_env/docker_buildcontext/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:20220418.v1
+
+RUN apt-get -y update && apt-get -y install wkhtmltopdf
+
+ENV AZUREML_CONDA_ENVIRONMENT_PATH /azureml-envs/responsibleai-0.18
+
+# Create conda environment
+RUN conda create -p $AZUREML_CONDA_ENVIRONMENT_PATH \
+    python=3.8 pip=21.3.1 -c anaconda -c conda-forge
+
+# Prepend path to AzureML conda environment
+ENV PATH $AZUREML_CONDA_ENVIRONMENT_PATH/bin:$PATH
+
+COPY requirements.txt /tmp/requirements.txt
+# Install pip dependencies
+# markupsafe and itsdangerous are bug workarounds
```

Comment:
There's a pip dependency resolver timeout issue that I need to test a fix for. If I find a fix we can switch to the curated rai environment and remove this.

Reply:
That would be nice

Reply:
resolved with converged rai environment